graphql/project_resolver.go

```diff
@@ -42,7 +42,7 @@ func (r *projectResolver) Patches(ctx context.Context, obj *restModel.APIProject
 	apiPatches := []*restModel.APIPatch{}
 	for _, p := range patches {
 		apiPatch := restModel.APIPatch{}
-		err = apiPatch.BuildFromService(p)
+		err = apiPatch.BuildFromService(p, false)
```

Comment:
So we do reference `projectIdentifier` in a few places. 
So it would need to be defined here. 
https://github.com/evergreen-ci/spruce/blob/6849093c411ac34c05b4e70d5b7e9b8f449a0863/src/gql/fragments/patchesPage.graphql#L6

Reply:
haha jinx